src/test/fileutils.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
```

Comment:
headerized these functions so I can reuse them easily in the pfu tests

Reply:
🖖🏻